mackerel-plugin-memcached/memcached_test.go

```diff
@@ -54,7 +54,9 @@ STAT conn_yields 1487476
 STAT bytes 621371972
 STAT curr_items 955652
 STAT total_items 2423543841
+STAT evicted_nonzero 10
```

Comment:
To obtain `evicted_nonzero` like Munin's memcached_multi_, at first the plugin should fetch output from `stat items`, and calculate the sum of each values for key `items:{slab}:evicted_nonzero`. (Munin's memcached_multi_ plugin does it at https://github.com/mhwest13/Memcached-Munin-Plugin/blob/master/memcached_multi_#L936-L941)